src/libinjector/win_injector.c

```diff
@@ -1660,26 +1665,34 @@ static event_response_t injector_int3_cb(drakvuf_t drakvuf, drakvuf_trap_info_t*
             return 0;
         }
 
-        access_context_t ctx = { 0 };
```

Comment:
These were specifically added to initialize the whole ctx structure to 0. Why are you removing it?

Reply:
This is a designated initializer:

struct_t s = { .a = 2, .b = 3 };

Other members are initialized as zero: "Omitted field members are implicitly initialized the same as objects that have static storage duration." (https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html)

So compiler do this for us.